packages/playground/src/weblets/full_vm.vue

```diff
@@ -168,33 +168,15 @@ const selectionDetails = ref<SelectionDetails>();
 const layout = useLayout();
 const tabs = ref();
 const solution = ref() as Ref<SolutionFlavor>;
-const images: VmImage[] = [
-  {
-    name: "Ubuntu-24.04",
-    flist: "https://hub.grid.tf/tf-official-vms/ubuntu-24.04-full.flist",
-    entryPoint: "",
-  },
-  {
-    name: "Ubuntu-22.04",
-    flist: "https://hub.grid.tf/tf-official-vms/ubuntu-22.04.flist",
-    entryPoint: "/init.sh",
-  },
-  {
-    name: "Ubuntu-20.04",
-    flist: "https://hub.grid.tf/tf-official-vms/ubuntu-20.04-lts.flist",
-    entryPoint: "/init.sh",
-  },
-  {
-    name: "Ubuntu-18.04",
-    flist: "https://hub.grid.tf/tf-official-vms/ubuntu-18.04-lts.flist",
-    entryPoint: "/init.sh",
-  },
-  {
-    name: "Nixos-22.11",
-    flist: "https://hub.grid.tf/tf-official-vms/nixos-22.11.flist",
-    entryPoint: "/init.sh",
-  },
-];
+const images: VmImage[] = Object.entries(Flists)
```

Comment:
Why should we do that while we know that we only need 5 images that was exist before

Reply:
Not extracting Flists like this will cause an error on images because of VMImage interface

Reply:
<img width="953" alt="image" src="https://github.com/user-attachments/assets/f589e25d-b17e-4335-9982-0dbc09087dcf" />


not sure what do you mean but this is my point we do not need to iterate over the object to just getting the already known flists 
```suggestion
const images: VmImage[] = [Flists.FULLVMS_NIXOS_22, Flists.FULLVMS_UBUNTU_18]
```

Reply:
I think after the latest flists refactor it's doable now